apps/mobile/src/app/wallet/wallet-settings/configure/[wallet]/index.tsx

```diff
@@ -0,0 +1,161 @@
+import { useEffect, useRef, useState } from 'react';
+import { ScrollView } from 'react-native-gesture-handler';
+import { useSafeAreaInsets } from 'react-native-safe-area-context';
+
+import { AddWalletModal } from '@/components/add-wallet/add-wallet-modal';
+import { Cell } from '@/components/cell';
+import { Divider } from '@/components/divider';
+import { RemoveWalletModal } from '@/components/wallet-settings/remove-wallet-modal';
+import { WalletNameModal } from '@/components/wallet-settings/wallet-name-modal';
+import { APP_ROUTES } from '@/constants';
+import { userRemovesWallet } from '@/state/global-action';
+import { useAppDispatch } from '@/state/utils';
+import {
+  WalletStore,
+  useWalletByFingerprint,
+  userRenamesWallet,
+} from '@/state/wallets/wallets.slice';
+import { BottomSheetModal } from '@gorhom/bottom-sheet';
+import { t } from '@lingui/macro';
+import { useTheme } from '@shopify/restyle';
+import { useLocalSearchParams, useNavigation, useRouter } from 'expo-router';
+import moment from 'moment';
+
+import {
+  ArrowOutOfBoxIcon,
+  ArrowsRepeatLeftRightIcon,
+  BarcodeIcon,
+  Box,
+  ChevronDownIcon,
+  ChevronUpIcon,
+  Eye1ClosedIcon,
+  InboxIcon,
+  SquareLinesBottomIcon,
+  Text,
+  Theme,
+  TouchableOpacity,
+  TrashIcon,
+} from '@leather.io/ui/native';
+
+function ConfigureWallet({ fingerprint, wallet }: { fingerprint: string; wallet: WalletStore }) {
+  const { bottom } = useSafeAreaInsets();
+  const theme = useTheme<Theme>();
+  const router = useRouter();
+  const addWalletModalRef = useRef<BottomSheetModal>(null);
+  const walletNameModalRef = useRef<BottomSheetModal>(null);
+  const removeWalletModalRef = useRef<BottomSheetModal>(null);
+  const [showMore, setShowMore] = useState(false);
+  const navigation = useNavigation();
+  const dispatch = useAppDispatch();
+  useEffect(() => {
+    navigation.setOptions({ title: wallet.name });
+  }, []);
+
+  function setName(name: string) {
+    dispatch(
+      userRenamesWallet({
+        fingerprint,
+        name,
+      })
+    );
+    navigation.setOptions({ title: name });
+  }
+
+  function removeWallet() {
+    router.back();
+    dispatch(userRemovesWallet({ fingerprint }));
+  }
+
+  return (
+    <>
+      <Box flex={1} justifyContent="space-between" backgroundColor="ink.background-primary">
+        <ScrollView
+          contentContainerStyle={{
+            paddingTop: theme.spacing['5'],
+            paddingBottom: theme.spacing['5'],
+            gap: theme.spacing[5],
+          }}
+        >
+          <Box px="5" gap="6">
+            <Text variant="heading05">{wallet.name}</Text>
+            <Cell
+              title={t`View Secret Key`}
+              Icon={Eye1ClosedIcon}
+              onPress={() => {
+                router.navigate({
+                  pathname: APP_ROUTES.WalletWalletsSettingsConfigureViewSecretKey,
```

Comment:
I branched off here to get started on `Sheet` and it seems like `APP_ROUTES.WalletWalletsSettingsConfigureViewSecretKey` doesn't exist?

Reply:
OK, maybe it's my mistake when re-basing and moving between branches 👍